src/components/Queue.vue

```diff
@@ -112,15 +98,14 @@ export default {
         },
     },
     mounted: function () {
-        if (Store.queue.sendingStatus) {
+        if (this.queue.sendingStatus) {
             this.startSpinner();
         }
     },
     computed: {
         currentlyShown: function () {
             return (
-                this.fail ||
-                (this.vouchers.length >= 1 && !Store.getVouchersOnlineStatus())
+                this.vouchers.length >= 1 && !Store.getVouchersOnlineStatus()
```

Comment:
Are you quite sure you don't need the brackets here?

Reply:
I think so; they were protecting the || and && conditions from each other.
without the || I think it's ok